sdk/storage/azblob/zc_blob_lease_client.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azblob
+
+import (
+	"context"
+	"errors"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/uuid"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+)
+
+type BlobLeaseClient struct {
+	BlobClient
+	LeaseID *string
+}
+
+// NewBlobLeaseClient is used to create lease client
+//nolint
```

Comment:
how does godoc treat the nolint, this might need to be at the end of the next line

Reply:
It works fine if declared before a function too.